pre_commit_hooks/detect_datetime_now.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import argparse
+import os
+
+
+def main(argv=None):
+    parser = argparse.ArgumentParser(description='Check if datetime.now is being used.')
+    parser.add_argument('filenames', nargs='*', help='Filenames to check')
+    args = parser.parse_args(argv)
+
+    for filename in args.filenames:
+        if os.path.isfile(filename):
+            with open(filename, 'rb') as f:
+                if b'datetime.now()' in f.read():
```

Comment:
This can be done even simpler as a [`pygrep`](https://pre-commit.com/#pygrep) hook

This would also give better output (currently the hook fails but doesn't mention the filename or line).

Reply:
Thank you. Your suggestion looks simpler.